src/components/battery/BatteryController.h

```diff
@@ -1,14 +1,42 @@
 #pragma once
 #include <cstdint>
 #include <drivers/include/nrfx_saadc.h>
+#include <array>
+#include <numeric>
 
 namespace Pinetime {
   namespace Controllers {
+  // A simple circular buffer that can be used to average 
+  // out the sensor values
```

Comment:
This should probably be in the Doxygen format so that it can be picked up by doc->HTML generators and IDEs
```
/* A simple circular buffer that can be used to average out the sensor values */
```